cmd/testpeers.go

```diff
@@ -144,50 +330,66 @@ func runTestPeers(ctx context.Context, w io.Writer, cfg testPeersConfig) (err er
 		}
 	}
 
+	done := make(chan os.Signal, 1)
+	signal.Notify(done, os.Interrupt)
```

Comment:
Maybe NotifyContext is better? https://github.com/pinebit/go-boot/blob/621b6b1a2df5d81ed9187219c08b9656cb2c0f05/boot/application.go#L39
(no need to deal with channels, just use context)